### PR TITLE
Fix warning docs return type

### PR DESCRIPTION
**Pull request by gregra81**
### Description of the PR
If returnType is not provided, set the @return as void

### CODE REVIEW

File: modules/swagger-codegen/src/main/resources/objc/api-header.mustache [hunk 1]
```diff
@@ -21,8 +21,8 @@ extern NSInteger k{{classname}}MissingParamErrorCode;
 /// {{#allParams}}@param {{paramName}} {{description}}{{^required}} (optional){{/required}}{{#defaultValue}} (default to {{{.}}}){{/defaultValue}}
 /// {{/allParams}}{{#responses}}
 ///  code:{{{code}}} message:"{{{message}}}"{{#hasMore}},{{/hasMore}}{{/responses}}
-///
-/// @return {{{returnType}}}
+///{{#returnType}}
+/// @return {{{returnType}}}{{/returnType}}
```
Comment (wing328) [inline modules/swagger-codegen/src/main/resources/objc/api-header.mustache]: For your upcoming PR, you may consider using the following style instead:
```
{{#returnType}}
/// @return {{{returnType}}}
{{/returnType}}
```

**Comment by wing328** [comment]
@gregra81 thanks for the PR.

Test results look good. Thanks for the contribution.

(If your company is using Swagger Codegen, please consider adding your company name to [this list](https://github.com/swagger-api/swagger-codegen#companiesprojects-using-swagger-codegen))

File: modules/swagger-codegen/src/main/resources/objc/api-header.mustache (hunk 1 above)
Comment (wing328) [inline modules/swagger-codegen/src/main/resources/objc/api-header.mustache]: @gregra81 I've submitted https://github.com/swagger-api/swagger-codegen/pull/4437 for the enhancement.

If you find other warnings with the ObjC client, please open a new "issue" for tracking.

**Comment by gregra81** [comment]
@wing328 - Thanks

**Pull request merged**